### PR TITLE
Add examples for Regulatory Compliance

### PR DESCRIPTION
**Pull request by dhh1128**
Signed-off-by: dhh1128 <daniel.hardman@sicpa.com>

***
<a href="https://pr-preview.s3.amazonaws.com/dhh1128/did-rubric/pull/35.html" title="Last updated on Aug 16, 2021, 6:07 PM UTC (c99b035)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-rubric/35/6e11c23...dhh1128:c99b035.html" title="Last updated on Aug 16, 2021, 6:07 PM UTC (c99b035)">Diff</a>

### CODE REVIEW

**Review by jandrieu — APPROVED** [review]

**Comment by jandrieu** [comment]
@dhh1128 Can you take a look at the merge conflicts?

**Comment by jandrieu** [comment]
FWIW, I merged the PRs in order, up to this one. Then I tried #36 and it worked fine.

So, this is the last one of the set.

**Comment by dhh1128** [comment]
Yep, I'll fix this.

File: index.html [hunk 1]
```diff
@@ -3301,49 +3301,69 @@ <h5>Question</h5>
...
                         </li>
                     </ol>
                 </section>
                 <section>
                     <h5>Relevance</h5>
-                    <p>
-                        ?
+                    <p>Some governments and enterprises may require DID methods that are
+                        officially endorsed by the EU, by their own security bureau, etc.
+                        Note that being out of harmony with government requirments may be
```
Comment (TallTed) [inline index.html:3326]:
```suggestion
                        Note that being out of harmony with government requirements may be
```

File: index.html [hunk 2]
```diff
@@ -3301,49 +3301,69 @@ <h5>Question</h5>
...
-                                By design, did:foo supports 123.5 bits of security.
+                                did:pkh
+                            </td>
+                            <td>
+                                C (for a reviewer requiring compliance with the Chinese
+                                government's SM2 cryptosystem); B (for a reviewer wanting to
+                                use the system in Canadian government contexts).
+                            </td>
+                            <td>
+                                The did:phk method subsumes multiple blockchain ecosystems. Most
```
Comment (TallTed) [inline index.html:3363]:
```suggestion
                                The `did:pkh` method subsumes multiple blockchain ecosystems. Most
```

**Review by jandrieu — APPROVED** [review]

**Comment by jandrieu** [comment]
It may be easiest just to reset --hard and manually cherry pick the changed sections.

**Comment by dhh1128** [comment]
I couldn't figure out why git kept interpreting this change as having a thousand-line merge conflict. So I re-created the change from scratch to make a nice, clean, small update. This PR is now obsoleted by #37.

**Pull request closed without merging**